docs/sdkx-unreal/delegates.mdx

```diff
@@ -101,7 +135,43 @@ This event is triggered when the user adds a message in a conversation. It might
 - Event name: `HelpshiftEventMessageAdd`
 - Event data keys:
   - `HelpshiftEventDataMessageType` - `HelpshiftEventDataMessageTypeAttachment` or `HelpshiftEventDataMessageTypeText`
-  - `HelpshiftEventDataMessageBody (string)`
+  - `HelpshiftEventDataMessageBody (FString)`
+
+### Agent Message Received Event {#agent-message-received}
+
+This event is triggered when the user receives any message from an agent in a conversation. This delegate is not triggered for bot messages or messages sent via automations.
+
+- Event name : `HelpshiftEventAgentMessageReceived`
+- Event data:
+
+  | Key                 | Key ( Raw ) | Type   | Value |
+  |-------------------------------|--------|--------|--------- |
+  | HelpshiftEventDataPublishID | publishId | FString | Conversation ID of the ongoing issue |
+  | HelpshiftEventDataMessageType | type | FString | Message Type of the message |
+  | HelpshiftEventDataMessageBody | body | FString | The actual message sent by the agent or empty |
+  | HelpshiftEventDataCreatedTime | createdTs | int32 | Unix epoch timestamp in ms |
+  | HelpshiftEventDataAttachments | attachments | Map<FString, Object>> | Attachments, if they are present |
+  | HelpshiftEventDataURL | url | FString | URL of the attachment |
+  | HelpshiftEventDataContentType | contentType | FString | MIME type of the attachment |
+  | HelpshiftEventDataFileName | fileName | FString | File name of the attachment |
+  | HelpshiftEventDataSize | size | int32 | Size of the attachment in bytes |
+  | HelpshiftEventDataMessageTypeAppReviewRequest | app_review_request | FString | Message type when agent request for App Review |
+  | HelpshiftEventDataMessageTypeScreenshotRequest | screenshot_request | FString | Message type when agent request screen shot |
+  | HelpshiftEventDataMessageTypeText | type | FString | Message type of text|
+ 
+<Admonition type="info" title="Note"> 
+
+- For Android , this delegate is available from 10.3.0 and above versions
```

Comment:
```suggestion
- This delegate is available from 10.3.0 onwards.
```

Reply:
This delegate is OTA for iOS but requires native changes that is 10.3.0 for android